deploy-manage/production-guidance/kibana-reporting-production-considerations.md

```diff
@@ -1,19 +1,21 @@
 ---
 navigation_title: "Reporting"
+mapped_pages:
+  - https://www.elastic.co/guide/en/kibana/current/reporting-production-considerations.html
+applies_to:
+  deployment:
+    self: all
 ---
 
-# Reporting production considerations [reporting-production-considerations]
+# Kibana reporting production considerations [reporting-production-considerations]
```

Comment:
the prod part of this is not so clear (chromium sandbox for secutity) - we should make that clearer in the overview.